webpack.config.development.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable */
+
+var path = require("path");
+var webpack = require("webpack");
+var assign = require("object-assign");
+
+var config = require("./webpack.config");
+
+config.devtool = 'cheap-module-eval-source-map';
+
+config.output = assign(config.output, {
+  publicPath: "/"
+});
+
+config.plugins = config.plugins.concat([
+  new webpack.NoEmitOnErrorsPlugin()
+]);
+
+config.module.rules[0] = {
```

Comment:
Don't like that this requires the first item to always be the `js|jsx` loader. Open to other approaches...